tests/helpers/test_update_coordinator.py

```diff
@@ -608,6 +617,26 @@ async def test_async_config_entry_first_refresh_success(hass: HomeAssistant) ->
     crd.setup_method.assert_called_once()
 
 
+async def test_async_config_entry_first_refresh_invalid_state(
+    hass: HomeAssistant,
+) -> None:
+    """Test first refresh successfully."""
+    entry = MockConfigEntry()
+    crd = get_crd(hass, DEFAULT_UPDATE_INTERVAL, entry)
+    crd.setup_method = AsyncMock()
+    with pytest.raises(
```

Comment:
Should this have a test when it doesn't raise?

Reply:
> Should this have a test when it doesn't raise?

We have already : test_async_config_entry_first_refresh_success

Reply:
Ah ok!
Test added